velociraptor/autoplotter/objects.py

```diff
@@ -662,13 +665,30 @@ def _add_lines_to_axes(self, ax: Axes, x: unyt_array, y: unyt_array) -> None:
         Adds any lines present to the given axes.
         """
 
+        # Fetch the minimum number of points to highlight
```

Comment:
Not sure if this is the best place to get this attribute. Perhaps, it is better to move these lines to `_parse_line()`, no?

Reply:
Yes, do all the parsing in _parse_line() before you use the attributes.